packages/vscode-extension/src/plugins/network/network-plugin.ts

```diff
@@ -1,18 +1,54 @@
 import http, { Server } from "http";
+import { spawn } from "child_process";
+import path from "path";
 import { commands, Disposable, window } from "vscode";
 import { WebSocketServer, WebSocket } from "ws";
 import { Devtools } from "../../project/devtools";
 import { ToolKey, ToolPlugin } from "../../project/tools";
 import { extensionContext } from "../../utilities/extensionContext";
+
+import { Logger } from "../../Logger";
 import { NetworkDevtoolsWebviewProvider } from "./NetworkDevtoolsWebviewProvider";
 
 export const NETWORK_PLUGIN_ID = "network";
 
+function startViteServer(): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const process = spawn("npm", ["run", "watch:network-webview"], {
```

Comment:
was this added just so we can have both extension Vite instance and network Vite running in parallel?

if so, then we can just use [concurrently](https://www.npmjs.com/package/concurrently)

```
concurrently "npm run watch:extension" "npm run watch:network-webview"
```

we used to use `run-run-all` in react-navigation for this purpose but the project is now abandoned